datastore/database/README.md

```diff
@@ -0,0 +1,10 @@
+# Database logs
+Set UID and GID of the postgres user
+`sudo chown -R 101:103 ./logs`
```

Comment:
Where/when is this `logs` directory created? I can't find it on my machine (after running `just local test`).

Reply:
This is no longer applicable. I forgot to remove this readme. It is now removed.